ai_genomics/utils/patent_data/get_ai_genomics_patents_utils.py

```
@@ -0,0 +1,84 @@
+"""Script to instantiate Google BigQuery client and
+functions to create bespoke genomics patents Google BigQuery
+tables in local project.
+"""
+######################################
+from google.cloud import bigquery
+from google.oauth2 import service_account
+import os
+
+######################################
+
+
+def est_conn():
+    """Instantiate Google BigQuery client
+    to query patent data.
+    """
+    if "GOOGLE_APPLICATION_CREDENTIALS" in os.environ:
+        google_creds = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")
+
+        credentials = service_account.Credentials.from_service_account_file(
+            google_creds
+        )
+
+        client = bigquery.Client(credentials=credentials)
+
+        return client
+
+    else:
+        print(
+            "export GOOGLE_APPLICATION_CREDENTIALS directory path as global variable."
+        )
```

Maybe raise a file not found error here instead?